### PR TITLE
chore: Add sponsors section to readme

### PR DESCRIPTION
**Pull request by FabianLars**
Should this be added to all plugin readmes too so it shows up on crates.io / npm too?

cc @nothingismagick

### CODE REVIEW

**Comment by FabianLars** [comment]
Actually yes, with the mirrors in place this should be in all readmes. gonna add that in a few

File: plugins/authenticator/README.md [hunk 1]
```diff
@@ -103,6 +103,22 @@ if (counter && counter > 0) {
...
 PRs accepted. Please make sure to read the Contributing Guide before making a pull request.
 
+## Partners
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="middle">
+        <a href="https://crabnebula.dev" target="_blank">
+          <img src="/.github/sponsors/crabnebula.svg" alt="CrabNebula" width="283">
```
Comment (amrbashir) [inline plugins/authenticator/README.md]: how will this be reflected in the mirror? does it copy the `.github` directory from here to the mirror?

Comment (FabianLars) [reply plugins/authenticator/README.md]: ffs good call, forgot about that

Comment (FabianLars) [reply plugins/authenticator/README.md]: ah damn, this also means the ~~tauri/~~ wry/tao readmes will be broken on crates.io right?

edit: tauri only has it in the repo root, not the crate readme...

Comment (amrbashir) [reply plugins/authenticator/README.md]: it should work in tao and wry, the `.github` directory is included, we have a splash file

Comment (FabianLars) [reply plugins/authenticator/README.md]: phew 😌

Comment (FabianLars) [reply plugins/authenticator/README.md]: okay the published v2 plugins already use a https link so i'll do the same. then it works for the mirrors and the package regs.

**Review by amrbashir — DISMISSED** [review]

**Comment by FabianLars** [comment]
i kinda dislike the auto review dismissal 😅

**Review by amrbashir — APPROVED** [review]

**Comment by FabianLars** [comment]
thanks :)

**Pull request merged**